app/components/candidate_interface/complete_references_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_with model: section_complete_form, url: candidate_interface_new_references_complete_path, method: :patch do |f| %>
+  <%= f.govuk_error_summary %>
```

Comment:
The error summary needs to appear above the h1 on the page, instead of between the references and the form (so that it's more visible on the page load).

```suggestion
```